build/shared/examples/11.ArduinoISP/ArduinoISP/ArduinoISP.ino

```diff
@@ -53,6 +53,22 @@
 #define SPI_CLOCK 		(1000000/6)
 
 
+// With an Genuino/Uno board, uncomment following line to generate a 4 MHz clock
+// signal on pin 3 (only available on Uno / ATmega*8)
+// 
+// Can be useful to recover an AVR chip with fuses setup for an external clock
+
+// #define ENABLE_CLOCK_GEN
+
+#ifdef ENABLE_CLOCK_GEN
+
+#if defined(__AVR_ATmega328__) || defined(__AVR_ATmega328P__) || defined(__AVR_ATmega168__) || defined(__AVR_ATmega88__)
```

Comment:
ArduinoISP sketch is meant to run on all Arduinos (also the ones with ARM chips), so it would be nice to keep the sketch functionalities in sync between all the targets.